Source/WebGPU/WebGPU/BindGroupLayout.mm

```diff
@@ -232,11 +231,18 @@ static bool containsStage(WGPUShaderStageFlags stageBitfield, auto stage)
         if (!descriptor)
             return BindGroupLayout::createInvalid(*this);
 
+        BindGroupLayout::BufferInfo bufferInfo;
+
         for (size_t stage = 0; stage < stageCount; ++stage) {
-            if (containsStage(entry.visibility, stage))
+            if (containsStage(entry.visibility, stage)) {
+                bufferInfo.bufferIndex[static_cast<ShaderStage>(stage)] = arguments[stage].count;
                 addDescriptor(arguments[stage], descriptor);
+            } else
+                bufferInfo.bufferIndex[static_cast<ShaderStage>(stage)] = BindGroupLayout::NotFound;
         }
```

Comment:
I will pull it out into a method if still needed after I take the suggestion from https://github.com/WebKit/WebKit/pull/14689/files#r1226056827

but then I think it will simply become:
```
bufferInfoForBinding.add(entry.binding, { stage, arguments[stage].count } );
```
so maybe no method needed with that approach